src/transaction/_manager.py

```diff
@@ -220,74 +226,153 @@ def run(self, func=None, tries=3):
                     raise
 
 
-@implementer(ITransactionManager)
-class ThreadTransactionManager(threading.local):
-    """Thread-local
-    `transaction manager <transaction.interfaces.ITransactionManager>`.
+if USE_CONTEXTVAR:  # pragma: PY3
```

Comment:
If you switch the `if` order it might make the diff much more easy to read:

```python
if not USE_CONTEXTVAR:
   # old/current code that on the diff is only shown as being indented
else:
  # new code, that shows up as new code
```

otherwise a quick glimse at the code feels like the *new* code is actually the old one 😅